app.py

```diff
@@ -154,8 +158,8 @@ def main():
                     data["results"][file] = f.readlines()
                 data["results"][file] = [i.replace("file_to_check.py", file)[1:] for i in data["results"][file]]
 
-                ## Remove the errors and warnings to be ignored from config
-                ## Also remove other errors in case of diff_only = True
+                # Remove the errors and warnings to be ignored from config
+                # Also remove other errors in case of diff_only = True
 
```

Comment:
Please don't convert `##` to `#`. If you notice, I'm using it as headers like markdown.